soc/arm/silabs_exx32/efr32bg22/Kconfig.series

```diff
@@ -0,0 +1,30 @@
+# Silicon Labs EFR32BG22 (Blue Gecko) MCU
+
+# Copyright (c) 2021 Sateesh Kotapati
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_EFR32BG22
+	bool "EFR32BG22P Series MCU"
+	select ARM
+	select CPU_CORTEX_M33
+	select CPU_HAS_FPU
+	select CPU_HAS_ARM_MPU
+	select CPU_HAS_ARM_SAU
+	select ARMV8_M_DSP
+	select ARM_TRUSTZONE_M
+	select CLOCK_CONTROL
+	select SOC_FAMILY_EXX32
+	select HAS_SILABS_GECKO
+	select HAS_SWO
+	select CPU_CORTEX_M_HAS_SYSTICK
```

Comment:
This is not reuquired. It's added to the Cortex-M33 definition.

Reply:
Removed CPU_CORTEX_M_HAS_SYSTICK